filebeat/filebeat.reference.yml

```diff
@@ -1223,6 +1223,17 @@ output.elasticsearch:
 # how to install the dashboards by first querying Elasticsearch.
 #setup.dashboards.always_kibana: false
 
+# If true and Kibana is not reachable at the time when dashboards are loaded,
+# it will retry to reconnect to Kibana instead of exiting with an error.
+#setup.dashboards.retry.enabled
```

Comment:
If there are default values we normally put them into the reference doc, so here it would be `#setup.dashboards.retry.enabled: false`. Similar for the values below.